tests/integration/test_postgres_backend.py

```
@@ -0,0 +1,461 @@
+    await postgres_backend.save_run_end(run_id, end_data)
+
+    # Verify updated details
+    updated_details = await postgres_backend.get_run_details(run_id)
+    assert updated_details is not None
+    assert updated_details["status"] == "completed"
+    assert updated_details["execution_time_ms"] == 5000
+    assert updated_details["total_steps"] == 3
+
+
+@pytest.mark.asyncio
+async def test_postgres_backend_auto_migrate_disabled(postgres_backend: PostgresBackend) -> None:
```

Ruff (0.14.7)

Unused function argument: `postgres_backend`

(ARG001)